nimble/syscfg.yml

```diff
@@ -130,6 +130,25 @@ syscfg.defs:
             This enables LE Connection Subrating feature
         value: 0
 
+    BLE_ADDITIONAL_PHY:
```

Comment:
I'd name it simply 'BLE_PHY'

Reply:
why do we need separate syscfg? just enable 2m or coded or both.

Reply:
It's because it may be simplier to cut down code if neither is supported. So when you check something besides 1M is supported you do
```c
#if MYNEWT_VAL(BLE_PHY)
```
instead of 
```c
#if MYNEWT_VAL(BLE_PHY_2M) || MYNEWT_VAL(BLE_PHY_CODED)
```

Reply:
```
#define BLE_PHY_ENABLED (MYNEWT_VAL(BLE_PHY_2M) || MYNEWT_VAL(BLE_PHY_CODED))

...

#if BLE_PHY_ENABLED
#endif
```

alternatively smth like this should work

```
syscfg.defs.'BLE_PHY_2M || BLE_PHY_CODED':
    BLE_PHY: 1
```

we should avoid syscfg vals that do nothing except they have to be set because another syscfg is set

Reply:
Makes sense to me, I updated PR with second option. Config generates OK